build/full-bundle.ts

```diff
@@ -27,7 +27,11 @@ export const buildFull = (minify: boolean) => async () => {
         exposeFilename: false,
       }),
       commonjs(),
-      esbuild({ minify, sourceMap: minify }),
+      esbuild({
+        minify,
+        sourceMap: minify,
+        target: 'es2018',
```

Comment:
Is it risky that ew compile our code into es2018? Even though we don't support certain browsers but this could be risky to me

Reply:
It used to be `es2017`. I don't know what risks we will have.

Reply:
Let's use this for now see if there were any complains.

Reply:
If someone really wants to run on a lower version browser, please transform by yourself.
And if you use `vite`, it will automatically transform the syntax to the `target` you configured.
Furthermore, `es2018` has no `Spread properties` and `Rest properties` than `es2017`.

https://esbuild.github.io/content-types/#javascript